osler/workup/tests/test_scripts.py

```diff
@@ -0,0 +1,31 @@
+from django.test import TestCase
+from osler.workup.models import Workup
+from osler.workup.scripts import workup_date_checker
+from .factories import WorkupFactory
+import datetime
+
+class testScripts(TestCase):
+
+    fixtures = ['workup', 'core']
+
+    def test_workup_date_checker(self):
+        d1 = datetime.datetime(2019, 4, 13)
+        d2 = datetime.datetime.now() - datetime.timedelta(days=2)
+        d3 = datetime.datetime(2018, 4, 4)
+        
+        workup_one = WorkupFactory()
+        workup_two = WorkupFactory()
+        workup_three = WorkupFactory()
+
+        workup_one.written_datetime = d1
+        workup_two.written_datetime = d2
+        workup_three.written_datetime = d3
```

Comment:
you should be able to specify the written datetime when creating the workup.  If this is not already allowed, you should edit WorkupFactory to allow this